app/assets/stylesheets/style.css.scss

```diff
@@ -61,3 +61,7 @@ footer.page-footer {
   min-width: 800px; min-height: 600px;
   background: url(/assets/bg.jpg);
 }
+
+.blockquote-orange{
+  border-left-color: #ffa726;
```

Comment:
Color literals like `#ffa726` should only be used in variable declarations; they should be referred to via variable everywhere else.